en/docs/develop/components/api-proxies/policies.md

```diff
@@ -0,0 +1,88 @@
+
+6. From the **Policy List** panel on the right-hand side, select the policy you want to apply and click **Add**.
+
+   ![Select policy](../../../assets/img/api-proxies/policies/select-policy.png)
+
+7. Click **Save** to save the changes.
+
+## Configuring the policy to accept environment-specific parameters
+
+You can use policies that enable users to enter custom parameters and values. For example, Add Header policy requires the user to enter a header name and value when deploying the REST API Proxy. These parameters can be static values or environment-specific values. 
+
+Follow the steps below to add environment-specific values and policy parameters:
+
+1. Sign in to the Choreo Console at [https://console.choreo.dev](https://console.choreo.dev).
+2. From the component list, select and click on the REST API Proxy component you want to attach policies to.
+3. In the left navigation menu, click **Develop**  and then click **Policies**.
+4. From the list of resources, expand the resource you want to attach the policies to.
+5. Click on **Attach Policy** of the respective flow you want to attach the policy to (`Request`, `Response`, or `Error` flow).
+6. Add a policy that enables you to accept custom parameters and values from users. For example, the **Add Header** policy. 
+
+       ![Add header](../../../assets/img/api-proxies/policies/add-header.png){.cInlineImage-full}
+
+8. The UI will prompt to provide values for the required policy parameters. Providing the value in “${name}” format makes the value a ballerina configurable.
+
+       ![Configurable values](../../../assets/img/api-proxies/policies/configurable-values.png)
+
+9. Click **Add** and then click **Save** to save the changes.
+10. To deploy the component, from the left navigation menu, click  **Deploy**  and then click **Config & Deploy**. Now, when the interceptor app generation step is complete, you can provide the configurations. 
+11. When prompted, provide the value and click **Save & Deploy**.
+
+     ![Save and deploy values](../../../assets/img/api-proxies/policies/save-and-deploy.png){.cInlineImage-full}
+
+
+## Monitoring and troubleshooting the deployment
+
+You check logs at each stage of the deployment through the right-hand-side console. To open this console, click on a build from the **Build and Deployment History** in the **Build Area**.
+
+#### Code Generation
+
+This is the stage where the interceptor app is generated based on the mediation policies. In this stage, if there are any errors, the user can view them in the console. If the error is due to a user error, for example, an error related to the API definition, etc., based on the logs the user can fix the issue and retrigger a build.
+
+#### Build
+
+The build stage builds the interceptor app as a docker image and deploys it in the respective environment. For logs in this stage, you can refer to the Ballerina build logs and the Docker build logs. 
```

Comment:
Shall we use "application" instead of "app"?

Use capital "D" for"Docker".

For the last stage, can we say something like "For logs related to the build, you can refer build logs of both Ballerina and Docker."

Reply:
Done

Reply:
The D in "docker image" is still simple.